tokio/src/runtime/metrics/runtime.rs

```diff
@@ -447,3 +447,65 @@ impl RuntimeMetrics {
         self.handle.spawner.worker_local_queue_depth(worker)
     }
 }
+
+cfg_net! {
+    impl RuntimeMetrics {
+        /// Returns the number of file descriptors currently registered with the
+        /// runtime's I/O driver.
+        ///
+        /// # Examples
+        ///
+        /// ```
+        /// use tokio::runtime::Handle;
+        ///
+        /// #[tokio::main]
+        /// async fn main() {
+        ///     let metrics = Handle::current().metrics();
+        ///
+        ///     let n = metrics.io_driver_fd_count();
+        ///     println!("{} fds currently registered with the runtime's I/O driver.", n);
+        /// }
+        /// ```
+        pub fn io_driver_fd_count(&self) -> u64 {
```

Comment:
Instead of returning the count here, i would just expose the counter values. So: `fd_registered_count` and `fd_deregistered_count`.

Reply:
Updated